Pod/Classes/Cards/Expiry.swift

```diff
@@ -11,26 +11,30 @@ import Foundation
 private let gregorianCalendar = NSCalendar(calendarIdentifier: NSCalendarIdentifierGregorian)!
 
 /**
- A Credit Card Expiry date
+ A Credit Card Expiry date.
  */
 public struct Expiry: RawRepresentable {
 
+    /// An invalid expiry date. This date is set to 1/1/1970 and therefor will be shown as `expired`.
     static let invalid = Expiry(rawValue: NSDate(timeIntervalSince1970: 0))
 
     public typealias RawValue = NSDate
```

Comment:
In the case of RawValue, the documentation is already "inherited" from the protocol anyways (at least when looking at it in Xcode):
`The "raw" type that can be used to represent all values of Self.`
So far, there are no type aliases where this is not the case.